code-challenges.test.js

```diff
@@ -15,24 +15,77 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+describe("arrayContaining", () => {
+  const exceptedArray1 = ["yellow", "blue", "pink", "green"];
+  const exceptedArray2 = [
+    "saffron",
+    "aquamarine",
+    "periwinkle",
+    "indigo",
+    "ochre",
+  ];
+  it("A function that takes in an array, removes the first item from the array and shuffles the remaining items", () => {
+    expect(colors1).toEqual(expect.arrayContaining(exceptedArray1));
+    expect(colors2).toEqual(expect.arrayContaining(exceptedArray2));
+  });
+});
+
+const colors1 = ["purple", "blue", "green", "yellow", "pink"];
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+const colors2 = [
+  "chartreuse",
+  "indigo",
+  "periwinkle",
+  "ochre",
+  "aquamarine",
+  "saffron",
+];
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
 // b) Create the function that makes the test pass.
 
+const arrayContaining = (array) => {
+  array.shift();
+  return array.sort((a, b) => 0.5 - Math.random());
+};
+
 // Pseudo code:
+// Prompt: A function that takes in an array, removes the first item from the array and shuffles the remaining items.
+// Input: array
+// Output: array
+// *** Steps ***
+// 1. Created a ES6 arrow function.
+// 2. Create a parameter that represents an array
+// 3. Apply .shift() to the array to remove the first element -> this is a mutator.
+// 4. Apply .sort() on the array
+// - Return the Math.random() - 0.5 (creates a new random number) from comparing the sorted array.
+// 5. Return the newly sorted array.
```

Comment:
Outstanding work and pseudocode on this problem!  If you are looking for a deeper challenge, try comparing this approach to the Fisher-Yates algorithm.  It's kind of a fun rabbit hole!